.travis.yml

```diff
@@ -49,68 +45,93 @@ jobs:
       before_script:
         - *clear-test-app-cache
       script:
-        - *run-phpunit-tests
-        - *clear-test-app-cache
         - *run-behat-tests
         - *validate-openapi-v2-json
         - *validate-openapi-v2-yaml
         - *validate-openapi-v3-json
         - *validate-openapi-v3-yaml
 
-    - name: '(PHP 7.2)'
+    - name: 'Behat (PHP 7.2)'
       php: '7.2'
       before_install:
         - *enable-mongodb-php-extension
         - *disable-xdebug-php-extension
         - *disable-php-memory-limit
-        - *add-composer-bin-dir-to-path
       install:
         - *update-project-dependencies
       before_script:
         - *clear-test-app-cache
       script:
-        - *run-phpunit-tests
-        - *clear-test-app-cache
         - *run-behat-tests
         - *validate-openapi-v2-json
         - *validate-openapi-v2-yaml
         - *validate-openapi-v3-json
         - *validate-openapi-v3-yaml
 
-    - name: '(PHP 7.3)'
+    - name: 'Behat (PHP 7.3)'
       php: '7.3'
       before_install:
         - *enable-mongodb-php-extension
         - *disable-xdebug-php-extension
         - *disable-php-memory-limit
-        - *add-composer-bin-dir-to-path
       install:
         - *update-project-dependencies
       before_script:
         - *clear-test-app-cache
       script:
-        - *run-phpunit-tests
+        - *run-behat-tests
+        - *validate-openapi-v2-json
+        - *validate-openapi-v2-yaml
+        - *validate-openapi-v3-json
+        - *validate-openapi-v3-yaml
+
+    - name: 'Behat (PHP 7.4-rc)'
+      php: '7.4snapshot'
+      env: LEGACY=0
+      before_install:
+        - *enable-mongodb-php-extension
+        - *disable-xdebug-php-extension
+        - *disable-php-memory-limit
+      install:
+        - *update-project-dependencies
+      before_script:
         - *clear-test-app-cache
+      script:
         - *run-behat-tests
         - *validate-openapi-v2-json
         - *validate-openapi-v2-yaml
         - *validate-openapi-v3-json
         - *validate-openapi-v3-yaml
 
-    - name: '(PHP 7.3) (lowest dependencies)'
+    - name: 'Behat (PHP 7.3) (lowest dependencies)'
       php: '7.3'
       before_install:
         - *enable-mongodb-php-extension
         - *disable-xdebug-php-extension
         - *disable-php-memory-limit
-        - *add-composer-bin-dir-to-path
       install:
         - composer update --no-progress --no-suggest --prefer-stable --prefer-lowest --ansi
       before_script:
         - *clear-test-app-cache
       script:
-        - *run-phpunit-tests
+        - *run-behat-tests
+        - *validate-openapi-v2-json
+        - *validate-openapi-v2-yaml
+        - *validate-openapi-v3-json
+        - *validate-openapi-v3-yaml
+
+    - name: 'Behat (PHP 7.4-rc) (lowest dependencies)'
+      php: '7.4snapshot'
+      env: LEGACY=0
```

Comment:
@dunglas Is this okay? At least until https://github.com/michelf/php-markdown/pull/316 is released?